home/.vimrc

```diff
@@ -98,6 +98,13 @@ map \ :NERDTreeToggle<CR>
 map <Leader>p gg:s/\s*$<CR>/\d\+-<CR>ywggA [#<esc>pa]<esc>
 nmap <Leader>l iLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<esc>
 
+
+" Turn `x = expr` into `let(:x) { expr }`, via comma-"let"
+nmap <Leader>let Ilet(:<esc>ea)<esc>f=r{A }<esc>
+
+" Select some ruby, comma-"var", and it'll extract a local variable.
+vmap <Leader>var cyour_variable<esc>Oyour_variable = <esc>p0*
+
```

Comment:
last updated 2011 :(

2015-03-12 14:18 GMT-04:00 Ben Oakes <notifications@github.com>:

> In home/.vimrc
> <https://github.com/ContinuityControl/dotfiles/pull/6#discussion_r26329423>
> :
>
> > @@ -98,6 +98,13 @@ map \ :NERDTreeToggle<CR>
> >  map <Leader>p gg:s/\s*$<CR>/\d\+-<CR>ywggA [#<esc>pa]<esc>
> >  nmap <Leader>l iLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<esc>
> >
> > +
> > +" Turn `x = expr` into `let(:x) { expr }`, via comma-"let"
> > +nmap <Leader>let Ilet(:<esc>ea)<esc>f=r{A }<esc>
> > +
> > +" Select some ruby, comma-"var", and it'll extract a local variable.
> > +vmap <Leader>var cyour_variable<esc>Oyour_variable = <esc>p0*
> > +
>
> Looks like vim-ruby-refactoring
> <https://github.com/ecomba/vim-ruby-refactoring> is a good candidate. I
> think we should try it out, especially before we put much more work into
> our own tooling for this.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuityControl/dotfiles/pull/6/files#r26329423>.
>



-- 
Joel Nimety
Senior VP, Engineering
Continuity
203.285.8133 (office)
203.763.9263 (cell)
http://www.continuity.net
http://www.linkedin.com/in/jnimety

-- 


-- 

------------------------------
This email and all files transmitted with it are intended solely for use by 
the intended recipient.  Any other use is prohibited.  If you received this 
email in error, please notify the sender, delete the email and destroy all 
attachments.  Opinions and ideas expressed herein are solely those of the 
sender, and may not reflect the official position of Continuity.